pkgs/development/arduino/arduino-core/default.nix

```diff
@@ -1,29 +1,35 @@
-{ stdenv, fetchFromGitHub, jdk, jre, ant, coreutils, gnugrep, file, libusb
-, withGui ? false, gtk2 ? null
+{ stdenv, fetchFromGitHub, jdk, jre, ant, coreutils, gnugrep, file, libusb, unzip,
+  gccRaw, readline, zlib, ncurses, withGui ? false, gtk2 ? null
```

Comment:
I don't see why you have ``gccRaw = gcc.cc``. Why not simply use ``stdenv.cc`` instead?